app/Repositories/Three/SignupRepository.php

```diff
@@ -17,19 +17,23 @@ public function create($data)
         // Create the signup
         $signup = new Signup;
 
-        $signup->northstar_id = $data['northstar_id'];
+        // @TODO change auth()->id(); based on Dave's note:
```

Comment:
Can we just do this in this PR?

Reply:
ah I forgot about this note to myself! I wanted to do this in this PR but wanted to do after I got everything working. Thanks for flagging, this @sbsmith86 !